Gemfile

```diff
@@ -8,6 +8,8 @@ gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 4.1'
 gem 'bootsnap', '>= 1.4.2', require: false
 gem 'rack-cors'
+gem 'rest-client'
+gem 'active_model_serializers'
```

Comment:
Did you use this gem somewhere? I couldn't find it

Reply:
The implementation was removed last minute, should thus be removed actually